.github/workflows/maven.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - axon-server-api-*.*.x
+      - persisted-streams
```

Comment:
I don't think we need this in the PR, right? Or will the `persisted-streams` branch live forever?